index.html

```diff
@@ -3,5 +3,4 @@
 ---
 
 {% include home/recent-projects.html %}
-{% include home/creative-portfolios.html %}
```

Comment:
You can remove the file in `_includes/home` too if you think we won't use it again.